Android/What is savedInstanceState bundle.md

```diff
@@ -0,0 +1,39 @@
+
+ 1. **Bundle savedInstanceState**  is a reference to the Bundle object that is passed to the `onCreate()` in an Activity. Activities have capability to restore themselves to a previous state using the stored in the Bundle.
+ 2. First time the `bundle` object is always null. If some data is stored in the bundle object during the Activity lifecycle and Activity is recreated due to rotation then the `bundle` object will be non null.
+ 
+ 
+During the  Activity Lifecycle save the data in `onSaveInstanceState()`
+```
+int scrolledPosition  = 0;
+ @Override
+ public void onSaveInstanceState(Bundle outState) {
+    super.onSaveInstanceState(outState);
+    outState.putInt("scrolled_position",scrolledPosition);
+ }
+```
+After the orientation when the Activity is recreated and onCreate() is called
+
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    
+        if(savedInstanceState == null){
+            scrolledPosition = 0;
+        }else{
+            scrolledPosition = savedInstanceState.getInt("scrolled_position",0);
+        }
+    }
+
+**What kind of data should be stored in `bundle` object**
+
+ 3. User selections
```

Comment:
Why this number list starts from 3?) Maybe I missing something, but I guess here we should start from 1, or we should change order of the blocks (block with information about what kind of data shouldn't stored in bundle should be first in this variant)